docs/synapse/userguides/syn_tools_feed.ipynb

```diff
@@ -11,7 +11,235 @@
     "Synapse Tools - feed\n",
     "====================\n",
     "\n",
-    "TBD"
+    "The Synapse ``feed`` tool is a way to ingest data exported from one Cortex into another Cortex. Users should be familiar with both the Synapse data model (:ref:`data-model-terms` et al.) as well as the Synapse concepts such as packed nodes and splices in order to use and understand the ``feed`` tool effectively.\n",
+    "\n",
+    "\n",
+    "Syntax\n",
+    "------\n",
+    "The ``feed`` tool is exected from an operating system command shell. The command usage is as follows (line is wrapped for readability):\n",
+    "\n",
+    "::\n",
+    "\n",
+    "  usage: synapse.tools.feed [-h] (--cortex CORTEX | --test) [--debug] [--format FORMAT] [--modules MODULES]   \n",
+    "    [--chunksize CHUNKSIZE] [--offset OFFSET] [files ...]\n",
+    "\n",
+    "Where:\n",
+    "- ``-h`` displays detailed helped and these command line options\n",
+    "- ``CORTEX``  specifies the path to the Cortex where the data should be ingested.\n",
+    "\n",
+    "  - See :ref:`cortex-connect` for the format to specify a path to a Cortex.\n",
+    "  - Cortex paths can also be specified using aliases defined in the user's ``.syn/aliases.yaml`` file.\n",
+    "\n",
+    "- ``--test`` means to perform ingest against a temporary, local Cortex instead of a live cortex, for test or validation\n",
+    "  \n",
+    "  - When using a temporary Cortex, you do not need to provide a path.\n",
+    "  \n",
+    "- ``--debug`` specifies to drop into an interactive prompt to inspect the state of the Cortex post-ingest. \n",
+    "\n",
+    "  - See :ref:`_syn-ref-cmdr` for how to use the interactive prompt\n",
+    "  \n",
+    "- ``FORMAT`` specifics the format the input files are in. \n",
```

Comment:
"specifics the format..." -> "specifies the format..."
"...the format the input files are in" -> "...the format of the input files."